src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -361,14 +408,24 @@ private static void setHeaders(Request sparkRequest, HttpRequestBase auth0Reques
     /**
      * Safely parse the userId and create an Auth0 url.
      *
-     * @param req The initating request that came into datatools-server
+     * @param req The initiating request that came into datatools-server
      */
     private static String getUserIdUrl(Request req) {
+        return getUserIdUrl(req, req.params("id"));
+    }
+
+    /**
+     * Safely parse the userId and create an Auth0 url.
+     *
+     * @param req The initiating request that came into datatools-server
```

Comment:
Here (and for `getAuth0UsersExcludeDefaultQuery`) there is an extra line in the javadoc. I don't think that's the convention elsewhere in this app.

Reply:
Updated.